### ISSUE TITLE
Implement fragment connect schema parser

### ISSUE THREAD

**Issue by iiscosd3** [tonkeeper/tonkeeper-web#36]
![image](https://github.com/tonkeeper/tonkeeper-web/assets/132331050/1c111c85-1614-4518-a84e-a7ce0c802390)
![image](https://github.com/tonkeeper/tonkeeper-web/assets/132331050/9fdaf753-a0db-4e6b-a19b-f0f9113fb7a9)
fails in there since its not implemented properly, works on android device though.

**Comment by iiscosd3** [comment]
Well, i just tried using it with a demo app (https://ton-connect.github.io/demo-dapp/) and it worked, why does it not work for fragment?
![image](https://github.com/tonkeeper/tonkeeper-web/assets/132331050/ebfd0e79-c3af-4d4a-910b-98cfc91db320)

**Comment by KuznetsovNikita** [comment]
> Well, i just tried using it with a demo app (https://ton-connect.github.io/demo-dapp/) and it worked, why does it not work for fragment? ![image](https://private-user-images.githubusercontent.com/132331050/305732697-ebfd0e79-c3af-4d4a-910b-98cfc91db320.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzNDM4OTcsIm5iZiI6MTcwODM0MzU5NywicGF0aCI6Ii8xMzIzMzEwNTAvMzA1NzMyNjk3LWViZmQwZTc5LWMzYWYtNGQ0YS05MTBiLTk4Y2ZjOTFkYjMyMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxOVQxMTUzMTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMDAwODg5MzQ5OGVlZTU1NTE2YzI4MGQwOTU4NzAwZTY1NjU1ZjE3ZWRhMTAyMzQ5MWFjY2NiOTQ2YjU5OGNjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.p1HiQEnWwkTG3TQfKNEjycWkNzaFRBiY_j3qSpjpv_A)

 Good day
 Fragment use a **TonConnect 1.0**, 
 it's a completely different protocol

**Comment by iiscosd3** [comment]
Yeah, i am aware, there's even a branch for it, right? I had some issues while building it, but merged some fixes from the current review to fix ts related compile time errors, yet the website doesn't work spewing some errors about webpack related modules and stuff like that (import outside of a module), anyway, what i was trying to ask for is a backport, as u can c theres a 'old' way/folder/implementation for older protocol in tonkeeper for android, is it possible to keep the backport for v1 in the current extension as well? at least on the basic level
